package/src/components/Channel/hooks/useTargetedMessage.ts

```diff
@@ -2,34 +2,41 @@ import { useEffect, useRef, useState } from 'react';
 
 export const useTargetedMessage = (messageId?: string) => {
   const clearTargetedMessageCall = useRef<ReturnType<typeof setTimeout>>();
-  const [targetedMessage, setTargetedMessage] = useState(messageId);
+  const [targetedMessage, setTargetedMessage] = useState<string | undefined>(messageId);
+  const [highlightedMessageId, setHighlightedMessageId] = useState<string | undefined>();
```

Comment:
Didn't we see some issue with this approach ? Or did it get resolved ?

Reply:
It was resolved